python/tests/test_unit_cosine_similarity.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+import numpy as np
+from pytest import raises
+from semantic_kernel.memory import VolatileMemoryStore
+from unittest import TestCase
+
+
+async def test_cosine_similarity_valid():
```

Comment:
Hey @vivekpandit-msft, are you able to run tests here? Have you done a `poetry install && poetry shell` and `pytest tests/`? 

When I do, I get the following: 
```
...
PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
  ...
```

I'm pretty sure this warning is coming from not marking these tests as `async` (you made the functions `async`, which was a necessary step, but we also need to tell `pytest` this info)

Reply:
Yes, after all your suggested changes it works